### ISSUE TITLE
Training: adjust loss function

### ISSUE THREAD

**Issue by mateuszmatu** [mateuszmatu/OceanAI#272]
We can play around with the loss function. 
Instead of just MSE, add different parameters. This would be a step towards physics informed NN. 
Eg: https://arxiv.org/abs/2501.19374
